src/middlewares/dataHubMiddleware.js

```diff
@@ -32,6 +33,13 @@ export function dataHubMiddleware(req, res, next) {
     // - With tracking consent, the client app should use a long lived cookie.
     const sessionToken = req.headers["x-session-token"];
 
+    // Visitor ip:
+    // Fingerprinting may cause collisions (same Visitor ID) - with visitor ip we reduce the risk.
+    // We forward the ip from proxy (@see proxy.js)
+    const ip = req.headers["x-forwarded-for"];
+    // we use MD5 for fast hashing - @TODO maybe we should use pseudonymizer??
+    const machineCode = ip ? Md5(ip) : "";
```

Comment:
Kan du ikke bruge vores utility-funktion createHash i stedet? md5 er usikkekr, og har større risiko for kollisioner.

Reply:
joda